blog/tests/views_test.py

```diff
@@ -0,0 +1,115 @@
+"""Tests for views' Blog app."""
+
+from datetime import datetime
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+
+from ..models import Post
+
+
+class PostListTest(TestCase):
+    """Tests for Post model."""
```

Comment:
Model tests belong to models test suite, not views.

Reply:
Understood